io.github.hakuneko.HakuNeko.json

```diff
@@ -0,0 +1,56 @@
+{
+    "app-id": "io.github.hakuneko.HakuNeko",
+    "base": "io.atom.electron.BaseApp",
+    "base-version": "stable",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "hakuneko",
+    "separate-locales": false,
+    "rename-desktop-file": "hakuneko-desktop.desktop",
+    "rename-icon": "hakuneko-desktop",
+    "finish-args": [
+		"--share=ipc",
+        "--socket=x11",
+        "--socket=pulseaudio",
```

Comment:
Nitpick: indentation

Reply:
I tried to fix the indentation of the script.